### ISSUE TITLE
Update calibrateArmDac.sh to create a $DATA_PATH/$DETECTOR/armDacCal folder

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/sw_utils#18]
## Brief summary of issue

It would be great if all log files and final calibration results where located in:

```bash
${DATA_PATH}/${DETECTOR}/armDacCal/scandate
```

Where `scandate` would be the datetime when the script either launched, or finished.

It should also make a `current` symlink. 

### Types of issue

- [ ] Bug report (report an issue with the code)
- [X] Feature request (request for change which adds functionality)

## Expected Behavior

Central location for output log files and calibration results as described above.

## Current Behavior

Output log files are located in the calling directory and the output calibration files are found in `$ELOG_PATH`.

## Context (for feature requests)

Quality of life feature.

**Comment by bdorney** [comment]
@AndrewLevin do you think you could tackle this?

The scurves that are launched should still be placed under `${DATA_PATH}/${DETECTOR}/scurve`

For an example see:


```bash
/data/bigdisk/GEM-Data-Taking/GE11_QC8/GE11-X-S-INDIA-0007/armDacCal% ll
total 8.0K
drwxrwxr-x. 2 gemuser gemuser 4.0K Feb  8 17:16 2019.02.02.03.51
lrwxrwxrwx. 1 gemuser gemuser   16 Feb  8 17:17 current -> 2019.02.02.03.51
/data/bigdisk/GEM-Data-Taking/GE11_QC8/GE11-X-S-INDIA-0007/armDacCal% ll 2019.02.02.03.51
total 7.6M
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 11:42 scurveLog_ArmDac_20.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 12:38 scurveLog_ArmDac_25.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 13:41 scurveLog_ArmDac_30.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 14:47 scurveLog_ArmDac_35.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 15:48 scurveLog_ArmDac_40.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 16:49 scurveLog_ArmDac_45.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 17:55 scurveLog_ArmDac_50.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 18:50 scurveLog_ArmDac_60.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 19:52 scurveLog_ArmDac_70.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 20:55 scurveLog_ArmDac_80.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 21:52 scurveLog_ArmDac_90.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 22:56 scurveLog_ArmDac_100.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  1 23:47 scurveLog_ArmDac_110.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  2 00:43 scurveLog_ArmDac_120.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  2 01:56 scurveLog_ArmDac_130.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  2 02:58 scurveLog_ArmDac_140.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  2 03:49 scurveLog_ArmDac_150.txt
-rwxrwxr-x. 1 gemuser gemuser  12K Feb  2 04:52 scurveLog_ArmDac_175.txt
-rwxrwxr-x. 1 gemuser gemuser  764 Feb  8 17:06 listOfScanDates_calibrateArmDac_GE11-X-S-INDIA-0007.txt
-rwxrwxr-x. 1 gemuser gemuser 367K Feb  8 17:14 canvScurveMeanByThrDac_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 326K Feb  8 17:14 canvScurveSigmaByThrDac_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 116K Feb  8 17:14 canvScurveMeanVsThrDac_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 123K Feb  8 17:14 canvScurveSigmaVsThrDac_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 127K Feb  8 17:14 canvScurveMeanVsThrDac_BoxPlot_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 110K Feb  8 17:14 canvScurveSigmaVsThrDac_BoxPlot_Summary.png
-rwxrwxr-x. 1 gemuser gemuser 6.2M Feb  8 17:14 calFile_CFG_THR_ARM_DAC_GE11-X-S-INDIA-0007.root
-rwxrwxr-x. 1 gemuser gemuser  874 Feb  8 17:14 calFile_CFG_THR_ARM_DAC_GE11-X-S-INDIA-0007.txt
-rwxrwxr-x. 1 gemuser gemuser 3.7K Feb  8 17:14 armDacCalLog_GE11-X-S-INDIA-0007.txt
```

**Comment by AndrewLevin** [comment]
Yes, I will do it.

**Comment by AndrewLevin** [comment]
Here is the pull request: https://github.com/cms-gem-daq-project/sw_utils/pull/19

Should I run a full test on one of the setups or should I hack a test that doesn't actually take new scurves?

**Comment by bdorney** [comment]
Resolved by #19
